docs/_static/network/v1.yaml

```diff
@@ -292,6 +292,10 @@ components:
         softwareVersion:
           description: Indication of the software version of the node. It's recommended to use a (Git) commit ID that uniquely resolves to a code revision, alternatively a semantic version could be used (e.g. 1.2.5).
           type: string
+        certificate:
+          description: >
+            Certificate presented during TLS handshake to setup the existing connection.
```

Comment:
This sounds convoluted (learned the word from @beardedfoo).

Just: the peer's certificate?